test/checkpoint/checkpointer.c

```diff
@@ -172,13 +185,18 @@ real_checkpointer(void)
         if (!g.running)
             goto done;
 
-        /* Verify the content of the checkpoint. */
-        if ((ret = verify_consistency(session, true)) != 0)
-            return (log_print_err("verify_consistency (offline)", ret, 1));
+        /* Verify the content of the checkpoint at the stable timestamp. */
+        if (g.use_timestamps)
+            if ((ret = verify_consistency(session, timestamp_buf)) != 0)
```

Comment:
We no longer verify here when not running with timestamps. Do we want to lose that functionality?
If we keep this logic, then you should remove a level of conditional and have it be:
if (g.use_timestamps && (ret = verify_...))

Reply:
I took that second stage of verification out because it felt like we were effectively doing the same thing with the online verification stage, I can put it back in if we feel like a 2x online verification is the way to go. We'll verify every time except after the final checkpoint in the non timestamped case.

Reply:
Ok, now I see. Please add a comment then (and fix the conditional) that says we can no longer validate the checkpoint unless timestamps are in use.